### PR TITLE
[W5.11r][W13-B1] Ang Jing Zhe

### PR DESCRIPTION
**Pull request by AJZ1995**
Add sort command for 2kLOC

### CODE REVIEW

File: test/java/seedu/addressbook/logic/LogicTest.java [hunk 1]
```diff
@@ -195,10 +195,34 @@ public void execute_list_showsAllPersons() throws Exception {
...
-                              true,
-                              expectedList);
+                Command.getMessageForPersonListShownSummary(expectedList),
+                expectedAB,
+                true,
+                expectedList);
+    }
+
+    @Test
+    public void execute_list_showAllPerson() throws Exception {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/logic/LogicTest.java:205]: 1. title of the test is incorrect. you are testing sort command and not list.
2. the test name doesn't follow the guidelines; please check the naming convention for tests in our coding standards.

File: test/java/seedu/addressbook/logic/LogicTest.java [hunk 2]
```diff
@@ -195,10 +195,34 @@ public void execute_list_showsAllPersons() throws Exception {
...
+
+        List<Person> expectedList = helper.generatePersonList(person1,person2,person3);
+        List<Person> unexpectedList = helper.generatePersonList(person2,person3,person1);
+
+        AddressBook expectedAB = helper.generateAddressBook(unexpectedList);
+
+        //prepare Addressbook state
+        helper.addToAddressBook(addressBook,unexpectedList);
+
+        assertCommandBehavior("sort",
```
Comment (okkhoy) [inline test/java/seedu/addressbook/logic/LogicTest.java:220]: This is a good "Positive" test. You can also consider writing tests to cover the negative scenarios.
While for this exercise there is no need to update any more tests, in your project you should definitely write tests covering both positive and negative cases.

**Comment by okkhoy** [comment]
@AJZ1995 
Some comments added. Please ack & close the PR after reading comments.

**Comment by AJZ1995** [comment]
Ack
